p2p/ipld/plugin/Makefile

```diff
@@ -1,34 +0,0 @@
-# Force Go Modules
-GO111MODULE = on
-
-GOCC ?= go
-GOFLAGS ?=
-
-# If set, override the install location for plugins
-IPFS_PATH ?= $(HOME)/.ipfs
-
-# If set, override the IPFS version to build against. This _modifies_ the local
-# go.mod/go.sum files and permanently sets this version.
-IPFS_VERSION ?= $(lastword $(shell $(GOCC) list -m github.com/ipfs/go-ipfs))
```

Comment:
I think in most cases, I manually had to set this to a local version of IPFS for this to work properly.

Reply:
Yeah, otherwise it wouldn't work with lazypatch, though I am not going to and don't see any reason to test it with IPFS to resolve the PR.

Reply:
That's OK.